CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 This is a list of high-level changes for each release of AWX. A full list of commits can be found at `https://github.com/ansible/awx/releases/tag/<version>`.
 
+## 12.0.0 (TBD)
+- Removed support for HipChat notifications ([EoL announcement](https://www.atlassian.com/partnerships/slack/faq#faq-98b17ca3-247f-423b-9a78-70a91681eff0))
```

Comment:
might want to mention that any hipchat stuff users have will be deleted. Just get that out in the open.